crates/iceberg-catalog/src/catalog/views.rs

```diff
@@ -141,12 +250,77 @@ impl<C: Catalog, A: AuthZHandler, S: SecretStore>
         )
         .await?;
 
-        return Err(ErrorModel::builder()
-            .code(StatusCode::NOT_FOUND.into())
-            .message("Views are not implemented".to_string())
-            .r#type("LoadViewNotSupported".to_string())
-            .build()
-            .into());
+        // ------------------- BUSINESS LOGIC -------------------
+        let namespace_id = C::namespace_ident_to_id(
+            warehouse_id,
+            &view.namespace,
+            state.v1_state.catalog.clone(),
+        )
+        .await?
+        .ok_or(
+            ErrorModel::builder()
+                .code(StatusCode::NOT_FOUND.into())
+                .message("Namespace does not exist".to_string())
+                .r#type("NamespaceNotFound".to_string())
+                .build(),
+        )?;
+
+        let Some(view_id) = view_id else {
+            return Err(ErrorModel::builder()
+                .code(StatusCode::NOT_FOUND.into())
+                .message(format!("View does not exist in warehouse {warehouse_id}"))
+                .r#type("ViewNotFound".to_string())
+                .build()
+                .into());
+        };
+        let mut transaction = C::Transaction::begin_read(state.v1_state.catalog).await?;
+
+        let GetWarehouseResponse {
+            id: _,
+            name: _,
+            project_id: _,
+            storage_profile,
+            storage_secret_id,
+            status,
+        } = C::get_warehouse(warehouse_id, transaction.transaction()).await?;
+        require_active_warehouse(status)?;
+
+        let view_metadata = C::load_view(view_id, transaction.transaction()).await?;
+
+        // TODO: should load_view return this? What's there to gain?
+        let location =
+            storage_profile.tabular_location(namespace_id, TabularIdentUuid::View(*view_id));
+        let metadata_location = storage_profile.metadata_location(&location, &view_metadata.uuid());
```

Comment:
This appears to clash with table renames across namespaces, should a rename trigger a metadata rewrite or should we violate the `{namespace_id}/{tabular_id}` path schema here? 

I'm leaning towards rewriting metadata on renames but happy to hear your opinion @c-thiel

Reply:
rewriting is not right, removed this and we're now returning the stored location.